ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -3616,25 +3616,6 @@ spec:
 ---
 apiVersion: v1
 kind: policy
-spec:
-  name: >
-    CIS - Ensure 'Windows Firewall: Public: Firewall state' is set to 'On (recommended)'
-  platforms: win10
-  platform: windows
-  description: |
-    Select On (recommended) to have Windows Firewall with Advanced Security use the settings for this profile to filter network traffic. If you select Off, Windows Firewall with Advanced Security will not use any of the firewall rules or connection security rules for this profile.
-    The recommended state for this setting is: On (recommended).
-  resolution: |
-    To establish the recommended configuration via GP, set the following UI path to On (recommended):
-    'Computer Configuration\Policies\Windows Settings\Security Settings\Windows Defender Firewall with Advanced Security\Windows Defender Firewall with Advanced Security - Local Group Policy Object\Windows Defender Firewall Properties\Public Profile\Firewall state'
-  query: |
-    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\WindowsFirewall\\PublicProfile\EnableFirewall' and data == 1);
-  purpose: Informational
-  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_9.3.9
```

Comment:
This query is indeed duplicate with 9.3.1
But looks like we are missing the original 9.3.9

FYI @RachelElysia This is the original:


`9.3.9 (L1) Ensure 'Windows Firewall: Public: Logging: Log dropped packets' is set to 'Yes' (Automated)
Profile Applicability:
 Level 1 (L1) - Corporate/Enterprise Environment (general use) Description:
Use this option to log when Windows Firewall with Advanced Security discards an inbound packet for any reason. The log records why and when the packet was dropped. Look for entries with the word DROP in the action column of the log.
The recommended state for this setting is: Yes. Rationale:
If events are not recorded it may be difficult or impossible to determine the root cause of system problems or the unauthorized activities of malicious users.
Impact:
Information about dropped packets will be recorded in the firewall log file.
Audit:
Navigate to the UI Path articulated in the Remediation section and confirm it is set as prescribed. This group policy setting is backed by the following registry location:
Remediation:
To establish the recommended configuration via GP, set the following UI path to Yes:
  HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\WindowsFirewall\PublicProfile\ Logging:LogDroppedPackets
 
Computer Configuration\Policies\Windows Settings\Security Settings\Windows Firewall with Advanced Security\Windows Firewall with Advanced Security\Windows Firewall Properties\Public Profile\Logging Customize\Log dropped packets
 438 | P a g e`

Reply:
This PR exposed the fact that we miss 9.3.9 but it did not do the actual harm so approving and openning a new bug to fix that in #13461